aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -25,64 +25,34 @@ import {
 import { getWidth } from './utils';
 
 export const defaultFormValues = {
+  'firstname-validation': '',
   'text-input-validation': '',
   'radio-button-group-validation': '',
   select: '',
   checkboxgroup: '',
   'text-area': '',
 };
 
-const stepOneValidate = values => {
-  const emailRegex = RegExp(
-    '[^@ \\t\\r\\n]+@[^@ \\t\\r\\n]+\\.[^@ \\t\\r\\n]+',
-  );
-  const emailValid = emailRegex.test(values['text-input-validation']);
-
-  return {
-    email: emailValid ? '' : 'Invalid email address.',
-    isValid: !!emailValid,
-  };
-};
-
-const validation = [
-  {
-    validator: values => stepOneValidate(values),
-    error: {
-      firstname: '',
-      lastname: '',
-      email: '',
-      radiobuttongroup: '',
-      isValid: true,
-    },
-  },
-];
-
 const stepOneInputs = [
-  (attemptedAdvance, error, formValues, setError) => (
+  (setValid) => (
     <>
       <FormField
         label="First Name"
         htmlFor="firstname-validation"
         name="firstname-validation"
-        error={error.firstname}
-        onChange={() =>
-          attemptedAdvance && setError(stepOneValidate(formValues))
-        }
+        required
```

Comment:
Wherever the required Form message gets set, can we set it to "This is a required field." to match Figma/form examples? As opposed to just "requried"

Reply:
Seems overly wordy. Is that the general UX guidance we want to promote?

Reply:
Added the required message like Form does. Thanks for the pointer.